src/Create.js

```diff
@@ -1,10 +1,85 @@
 import React, { Component } from 'react';
+import { getTypes, createDessert } from './fetch-utils.js';
+import classNames from 'classnames';
 
 // const URL = 'https://aqueous-coast-92629.herokuapp.com/desserts';
 
 class Create extends Component {
+    state = {
+            name: '',
+            icing: '',
+            type_id: 0,
+            message: '',
+            error: false,
+         };
+         componentDidMount = async () => {
+            const types = await getTypes();
+            this.setState({ types });
```

Comment:
Since you're setting `types` into state, you need to make sure that you have the key `types` initialized in your state object first. This way the function `setState` has a property to update when you call it.